earthpy/plot.py

```diff
@@ -327,7 +328,8 @@ def plot_rgb(
     ax.imshow(rgb_bands, extent=extent)
     ax.set_title(title)
     ax.set(xticks=[], yticks=[])
-    return fig, ax
```

Comment:
This seems like a substantial change in behavior - is this related to #282? 

If users are trying to use this function to make multipanel plots or use the `fig` object, will this change prevent them from doing so?

Reply:
Nope. 
1. you can always access the figure object using ax.figure if need be.
BUT if a user wants to implement multi-panel, they need to setup the plot themselves and then use the `ax=` parameter ... 

this test
```
def test_ax_provided(rgb_image):
```
tests that a user can in fact provide a custom axis and things still work!

Reply:
Sweet!